eth_utils/address.py

```diff
@@ -72,44 +74,41 @@ def is_32byte_address(value):
         return False
 
 
-@coerce_args_to_text
-def is_address(value):
+def is_address(primitive=None, hexstr=None, text=None):
```

Comment:
Hrm, not 100% sure this is going to work but, I think this should also be a single `value` parameter and we'll count on the `is_XX_address` functions to do the proper handling.

Reply:
@pipermerriam sorry, just want to make sure i've got this straight. `is_hex_address` should only accept `is_text` types, `is_binary_address` should only accept `is_bytes` types, and `is_32byte_address` should only accept `is_string` types ?? Or should all three only accept `is_text` types ??